lib/lib.vcxproj

```diff
@@ -0,0 +1,188 @@
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions>YY_USE_PROTOS;YY_NEVER_INTERACTIVE;__STDC_VERSION__=199901L;__STDC_WANT_SECURE_LIB__=1;_LITTLE_ENDIAN=1;BYTE_ORDER=_LITTLE_ENDIAN;_WINSOCK_DEPRECATED_NO_WARNINGS;_CRT_SECURE_NO_WARNINGS;_USRDLL;_CRT_NON_CONFORMING_SWPRINTFS;WIN32_LEAN_AND_MEAN=1;_WIN32_WINNT=0x0A00;WINVER=0x0A00;WINNT=1;NTDDI_VERSION=0x0A000000;_WINDOWS</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>$(ProjectDir)libctf\common;$(ProjectDir)libdtrace\common;$(ProjectDir)libdtrace\compat\win32;$(ProjectDir)libdtrace\compat\win32\inc;$(ProjectDir)libdtrace\compat\win32\prebuilt;$(ProjectDir)..\sys\dev\dtrace\x86;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>
+      <DisableSpecificWarnings>4274</DisableSpecificWarnings>
+      <RuntimeLibrary />
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>onecore_apiset.lib;dbghelp.lib;delayimp.lib</AdditionalDependencies>
+      <ModuleDefinitionFile>$(ProjectDir)libdtrace\compat\win32\dtrace.def</ModuleDefinitionFile>
+      <DelayLoadDLLs>ws2_32.dll;crypt32.dll</DelayLoadDLLs>
+      <IgnoreAllDefaultLibraries>true</IgnoreAllDefaultLibraries>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
+    <ClCompile>
+      <PreprocessorDefinitions>_DEBUG;DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>libvcruntimed.lib;libcmtd.lib;ucrtd.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
+    <ClCompile>
+      <PreprocessorDefinitions>_NDEBUG;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>libvcruntime.lib;libcmt.lib;ucrt.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <CustomBuild Include="libdtrace\common\dt_grammar.y">
+      <Command>"$(SolutionDir)\releng\external\tools\bin\win_bison.exe" -y "-o$(ProjectDir)\libdtrace\compat\win32\prebuilt\%(FileName).c" -d -y "%(FullPath)"</Command>
+      <Message>Bison: %(Identity)</Message>
+      <Outputs>$(ProjectDir)\libdtrace\compat\win32\prebuilt\%(Filename).c;$(ProjectDir)\libdtrace\compat\win32\prebuilt\%(Filename).h</Outputs>
+      <OutputItemType></OutputItemType>
+    </CustomBuild>
+    <CustomBuild Include="libdtrace\common\dt_lex.l">
+      <Command>"$(SolutionDir)releng\external\tools\bin\win_flex.exe" -l --wincompat "-o$(ProjectDir)libdtrace\compat\win32\prebuilt\%(FileName).c" "%(FullPath)"</Command>
+      <Message>Flex: %(Identity)</Message>
+      <Outputs>$(ProjectDir)libdtrace\compat\win32\prebuilt\%(Filename).c</Outputs>
```

Comment:
Would be great to use $(IntDir) here and in similar places instead to keep all autogenerated files in the same location.

Reply:
I think I went down that road initially, then ran into conflicts as dtrace.obj can ambiguously refer to dtrace.exe or dtrace.dll

Reply:
@ashedel (Oops, hit the wrong button.) Sounds good, will do now.